### PR TITLE
✨ Handle returning visitors to the CLI

### PR DESCRIPTION
**Pull request by scopsy**

### CODE REVIEW

File: packages/cli/src/commands/init.ts [hunk 1]
```diff
@@ -183,3 +194,59 @@ const keyPress = async (): Promise<void> => {
...
+        name: 'result',
+        message: `Looks like you already have a created an account for ${existingApplication.name}`,
+        choices: [
+          {
+            name: `Visit ${existingApplication.name} Dashboard`,
+            value: 'visitDashboard',
+          },
+          {
+            name: 'Create New Account',
+            value: 'newAccount',
```
Comment (djabarovgeorge) [inline packages/cli/src/commands/init.ts:228]:
```suggestion
            value: 'new',
```

**Review by djabarovgeorge — COMMENTED** [review]
Looks great! there was one line that I was not sure about it therefore I left a suggestion, and I do think that we can refactor handleExistingSession function in the future as it is a bit Inconclusive. On the one hand, she handles and on the other hand, she returns a boolean if to continue the process or not. In my opinion, it will be worth dividing the function a bit in the future.

**Comment by scopsy** [comment]
@djabarovgeorge had this hunch too, let me play with it to see if I can find better approach

**Comment by scopsy** [comment]
@djabarovgeorge Extracted the onboarding process to a different function and changed the current function structure. WDYT?

**Comment by djabarovgeorge** [comment]
Looks great! 🚀

**Review by djabarovgeorge — APPROVED** [review]

**Pull request merged**